README.md

```diff
@@ -12,7 +12,7 @@ CSI driver for provisioning Local PVs backed by LVM and more.
 
 ## Project Status
 
-Currently the LVM CSI Driver is in beta.
+LVM-LocalPV CSI Driver is declared GA in August 2021.
```

Comment:
Should we mention the version as well ? like `declared GA from [v0.8.0](url) released in August 2021`

Reply:
done @prateekpandey14